platform/src/main/java/org/stellar/anchor/platform/data/PaymentObservingAccountRepo.java

```diff
@@ -0,0 +1,8 @@
+package org.stellar.anchor.platform.data;
+
+import org.springframework.data.repository.CrudRepository;
+
+public interface PaymentObservingAccountRepo
```

Comment:
Is this interface actually implemented anywhere?

Reply:
The implementation is generated by Spring Data module. That means, only the interfaces are defined. Spring will generate database ORM code according to the names of the interface functions. 

Here is a good example of declaring JDBC repositories.
https://www.baeldung.com/spring-data-jdbc-intro#declaring-jdbc-repositories

Reply:
I missed the nuance that some of these classes are tables. That makes more sense!